drake/systems/plants/collision/FCLModel.h

```diff
@@ -0,0 +1,104 @@
+#ifndef __DrakeCollisionFCLModel_H__
```

Comment:
That is an illegal symbol because of the double underscore. [Here](http://stackoverflow.com/questions/228783/what-are-the-rules-about-using-an-underscore-in-a-c-identifier) is a good summary of the rules. I suggest this instead:
```c++
#ifndef DRAKE_COLLISION_FCL_MODEL_H_
```
which satisfies the language requirements and also the coding convention that macros should be in ALL_CAPS.